draft-ietf-grow-bmp-local-rib.xml

```diff
@@ -290,14 +290,16 @@
                     Requires additional BGP peering to collect the received updates
                     when peering may have not even been required in the first
                     place. For example, virtual routing and forarding (VRF) tables 
-                    with no peers, redistributed BGP-LS with no peers,
+                    with no peers, redistributed BGP-LS with no peers, and
                     segment routing egress peer engineering where no
-                    peers have link-state address family enabled.
+                    peers have link-state address family enabled are all
+                    situations with no preexisting BGP peer where Loc-RIB
```

Comment:
This looks like a good change, but "where Loc-RIB access via BMP would be useful." should be dropped.  The example is where we do not have preexisting peers and having to add one just to collect data is suboptimal and wasteful (IMHO).